package.json

```diff
@@ -455,6 +455,7 @@
     "vega": "^5.17.3",
     "vega-interpreter": "npm:@amoo-miki/vega-forced-csp-compliant-interpreter@1.0.6",
     "vega-lite": "^4.16.8",
+    "vega-lite-next": "npm:vega-lite@^5.6.0",
```

Comment:
What will it take for use to simply use v5 as the primary dependency for vega lite?

Reply:
Great question! It really is the same as any other direct dependency version bump we want to do that contains breaking changes. As of this PR, core dashboards no longer has any dependencies on v4. But I'm not sure what the campaign or deprecation process has been in the past to let plugin teams know that we'll be bumping vega-lite from 4 to 5 in OpenSearch Dashboards 3.0.

Separately, I feel like we also need a campaign to communicate the idea that as of 3.0, we'll no longer consider direct `package.json` dependencies as part of the public API subject to semver. Although maintainers have discussed that issue a lot, we probably need a proposal for outlining the impact of such a change.

Reply:
So are we saying this is a breaking change that we want people to migrate to for OpenSearch Dashboards 3.0.0?

Reply:
No, it's non-breaking - instead of updating `vega-lite` we aliased the v5 version to `vega-lite-next` and switched our internal usage and imports to that. But any plugin importing `vega-lite` still has access to the v4 version. (And vega-lite v5 also fully supports parsing v4 specs). But Ashwin's point is that we should plan for a more permanent change in the future so we're not just carrying around an old version that may not even be used.

Reply:
Isn't the timeline for 3.0 TBD? Why not start the process of communicating with plugins now and just rip off the metaphorical band-aid?

Reply:
Yeah, the ideal time to start communicating about being able to update dependencies outside major version updates was probably months ago. "Now" is probably next best 😁

Reply:
So can we just bump to v5 then and skip the alias?

Reply:
I agree. We have to stop bending over backwards fearing a major version upgrade like this. We can always find a mitigation but that just adds to the pain of releasing for 3.0 when we have to undo all these bandaids.

Reply:
No? Why would we arbitrarily change our semver definition now? If this is a straight version bump, any feature that depends on it will be blocked until a `3.0.0` release which isn't even scheduled yet. If the argument is that this PR (as-is) should target `2.x`, and that you would prefer a separate PR to main without the alias, I'm onboard with that.

Reply:
> As of this PR, core dashboards no longer has any dependencies on v4

Considering this statement, there is no reason for us to alias it. Our semver definition has always been about what we serve the end users: those who install OSD and those who consume it. Dependencies of OSD are bumped all the time; we don't hold back due to being considerate for plugins and they are forced to keep up due to the rigid version dependency matching logic.

Reply:
Updated this PR to remove alias - this is now a breaking change PR, targeting v3.0.0. I'll manually submit a backport with the alias for the `2.x` branch.